test/test_elbo_values.jl

```diff
@@ -86,7 +86,7 @@ function test_derivative_flags()
     elbo = DeterministicVI.elbo(ea)
 
     elbo_noderiv = DeterministicVI.elbo(ea; calculate_derivs=false)
-    @test_approx_eq elbo.v[1] elbo_noderiv.v
+    @test_approx_eq elbo.v[1] elbo_noderiv.v[1]
```

Comment:
Recall that when I went through the ELBO very carefully last semester looking for everywhere that memory was being allocated, every time the ```Float64``` value ```v``` was being set in a sensitive float, it was allocating memory.  Changing to an array prevented quite a lot of memory allocation.

I agree that this is strange and maybe it's not the case anymore.